coordinator/internal/stateguard/stateguard_test.go

```diff
@@ -33,26 +35,48 @@ contrast_coordinator_manifest_generation %d
 `
 )
 
-var keyDigest = manifest.HexString("0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef")
-
-func TestSNPValidateOpts(t *testing.T) {
+func TestMetrics(t *testing.T) {
```

Comment:
Should the test also cover case where state.CompareAndSwap in UpdateState fails?